test/integration/crcsuite/crcsuite.go

```diff
@@ -97,7 +97,7 @@ func FeatureContext(s *godog.Suite) {
 
 		// put CRC binary location on top of PATH
 		path := os.Getenv("PATH")
-		newPath := fmt.Sprintf("%s%s%s", CRCBinary, os.PathListSeparator, path)
+		newPath := fmt.Sprintf("%s%s%s", CRCBinary, string(os.PathListSeparator), path)
```

Comment:
have you tried `%c` and no cast?

Reply:
`%c` works too, yes. Is it better to do it that way?

Reply:
It seems less convoluted this way, we use the correct printf modifier for the type we want to print, rather than converting the value to string, and then printing it.